geom/src/polygon.rs

```diff
@@ -476,19 +476,37 @@ fn to_geo(pts: &Vec<Pt2D>) -> geo::Polygon<f64> {
     )
 }
 
-fn from_geo(p: geo::Polygon<f64>) -> Polygon {
-    Polygon::buggy_new(
-        p.into_inner()
-            .0
-            .into_points()
-            .into_iter()
-            .map(|pt| Pt2D::new(pt.x(), pt.y()))
-            .collect(),
-    )
+impl From<geo::Polygon<f64>> for Polygon {
+    fn from(poly: geo::Polygon<f64>) -> Self {
+        let (exterior, interiors) = poly.into_inner();
+        Polygon::with_holes(
```

Comment:
I wonder why I was using `buggy_new`.. a quick audit of `convex_hull()` and `intersection()` doesn't turn up anything that should be used in map importing, so I think this should work fine. Just never updated this call-site.